fastly/block_fastly_service_v1_acl_test.go

```diff
@@ -42,10 +43,18 @@ func TestResourceFastlyFlattenAcl(t *testing.T) {
 
 func TestAccFastlyServiceV1_acl(t *testing.T) {
 	var service gofastly.ServiceDetail
-	name := fmt.Sprintf("tf-test-%s", acctest.RandString(10))
+	var acl gofastly.ACL
+	name := acctest.RandomWithPrefix(testResourcePrefix)
 	aclName := fmt.Sprintf("acl_%s", acctest.RandString(10))
 	aclNameUpdated := fmt.Sprintf("acl_updated_%s", acctest.RandString(10))
 
+	// Six part test:
+	// 1. Create service with 2 ACLs
+	// 2. Rename both the ACLs, should succeed because the ACLs are empty
+	// 3. Keep both ACLs the same and add an entry to one of them
+	// 4. Try to rename the ACLs, expect to fail with "list not empty error"
+	// 5. Without renaming the ACLs, set force_destroy=true to skip the deletion check
+	// 6. Try to rename the ACLs again, expect to succeed
```

Comment:
❤️  the descriptive breakdown here. This has been lacking in the majority of the tests 👍🏻

Reply:
Thanks! Thought it was especially necessary given the longer-form test